File: src/main/java/com/wanderers/hotelier_webservices/server/exception/ResultNotFoundException.java
package com.wanderers.hotelier_webservices.server.exception;

public class ResultNotFoundException extends HotelierDaoException {

    private static final long serialVersionUID = 1L;

    /**
     * Class constructor
     *
     * @param message String value of Exception
     */
    public ResultNotFoundException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: src/main/kotlin/com/wanderers/hotelier_webservices/server/exception/ResultNotFoundException.kt
package com.wanderers.hotelier_webservices.server.exception

class ResultNotFoundException
/**
 * Class constructor
 *
 * @param message String value of Exception
 */
    (message: String?, cause: Throwable?) : HotelierDaoException(message, cause) {
    companion object {
        private const val serialVersionUID = 1L
    }
}